### PR TITLE
Rework post result

### PR DESCRIPTION
**Pull request by manasdk**
Instead of passing in an execution object gets the execution object in the action
to drop the dependency on the st2 pack from st2contrib.

### CODE REVIEW

**Comment by Kami** [comment]
That's what I suggested in the past when we did "pass in execution object instead of id" change so I'm +1 for this approach.

File: contrib/chatops/actions/format_execution_result.py [hunk 1]
```diff
@@ -42,3 +43,12 @@ def run(self, execution):
                     template = alias.result['format']
 
         return self.jinja.from_string(template).render(context)
+
+    def _get_execution(self, execution_id):
+        if not execution_id:
+            raise ValueError('Invalid execution_id provided.')
+        execution = self.client.liveactions.get_by_id(id=execution_id)
```
Comment (Kami) [inline contrib/chatops/actions/format_execution_result.py]: So is it retrieving an execution or live action?

This stuff still is confusing - either client manager name is incorrect or the method, argument, etc. here is incorrect.

Comment (Kami) [inline contrib/chatops/actions/format_execution_result.py]: I checked the code, looks like the client name is wrong.

Gosh, this is so annoying and confusing. Maybe to make it less confusing let's create new `Execution` resource class in `st2client/models/action|execution` and let's have `LiveAction` inherit from it and leave it there for backward compatibility reasons.

Comment (manasdk) [inline contrib/chatops/actions/format_execution_result.py]: I am ont touching client code right now. Certainly not in this PR.

**Pull request merged**